includes/utils.php

```diff
@@ -285,14 +285,28 @@ function available_pull_post_types( $connection, $type ) {
 /**
  * Return post types that are allowed to be distributed
  *
+ * @param string $output Optional. The type of output to return.
+ *                       Accepts post type 'names' or 'objects'. Default 'names'.
+ *
  * @since  1.0
  * @return array
  */
-function distributable_post_types() {
-	$post_types = get_post_types( [ 'public' => true ] );
+function distributable_post_types( $output = 'names' ) {
+	$post_types = get_post_types( [ 'show_in_rest' => true ], $output );
```

Comment:
```suggestion
	$post_types = get_post_types( [ 'show_in_rest' => true ] );
	$post_types = array_filter( $post_types, 'is_post_type_viewable' );
```

This will ensure only publicly queryable post types are shown -- ie, post types that can be seen by visitors on the front end of the site. 

See comment on filter for why I've removed the output type.